Assets/Scripts/AudioManager.cs

```diff
@@ -0,0 +1,37 @@
+using UnityEngine;
+
+public class AudioManager : MonoBehaviour
+{    
+    public AudioSource Walking, Jumping, Climbing, Win;
+
+    public void PlayWalking()
```

Comment:
Is there any reason why you made this functions public?

Reply:
Yes Sir, I am calling these functions in Player object. 
_Eg. If Player is walking, play walking sounds for AudioManager._

I created a separate script for audio functions only to organize them easier.